README.txt

```diff
@@ -101,6 +101,11 @@ If the recurring opportunity will have the same volunteers each time, we’d rec
 
 == Changelog ==
 
+= 1.3.13
```

Comment:
Just wanted to note here to remember to update this if we change the other code.

Reply:
This has been updated!